whist/server/api/game/create.py

```diff
@@ -44,6 +51,11 @@ def _set_game_parameter(request, user, pwd_service: PasswordService):
 
 
 def _get_amount_player(request, key) -> Optional[int]:
+    """
+    Checks to see if player is in the level range.
+    :param request: Must contain a 'game_name'. 'password' is optional
+    :param key: the player entering game
+    """
```

Comment:
Try to understand what this method actually does. We don't even have player level, but ratings.